### PR TITLE
Release/0.8.0

### PR DESCRIPTION
**Pull request by chuwy**
- [x] Rename https://github.com/snowplow-incubator/snowplow-snowflake-loader/issues/120 commit

### CODE REVIEW

File: loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/Statement.scala [hunk 1]
```diff
@@ -174,4 +174,9 @@ object Statement {
       SqlStatement(s"SHOW warehouses$schemaPattern")
     }
   }
+
+  implicit object DescriveTableStatement extends Statement[DescribeTable] {
```
Comment (oguzhanunlu) [inline loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/Statement.scala]:
```suggestion
  implicit object DescribeTableStatement extends Statement[DescribeTable] {
```

Comment (chuwy) [reply loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/Statement.scala]: Can't wait while instance identifiers become optional in Scala 3.

**Review by oguzhanunlu — APPROVED** [review]

File: core/src/main/scala/com.snowplowanalytics.snowflake.core/RunManifests.scala [hunk 2]
```diff
@@ -0,0 +1,242 @@
+/*
```
Comment (dilyand) [inline core/src/main/scala/com.snowplowanalytics.snowflake.core/RunManifests.scala:1]: Just curious: how is this change related to scala 2.12?

File: loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/AtomicDef.scala [hunk 3]
```diff
@@ -214,4 +217,31 @@ object AtomicDef {
...
+    columns
+      .zipAll(AtomicDef.columns.map(_.asRight), MissingExisting.asLeft, MissingAtomic.asLeft)
+      .zipWithIndex
+      .flatMap { case ((existing, atomic), i) =>
+        val position = i + 1   // indexes are 0-based
+        (existing, atomic) match {
+          case (Right(e), Right(a)) if e == a =>   // Match
+            Nil
+          case (Right(e), Right(a)) =>             // Different
+            List(s"Existing column [${e.show}] doesn't match expected definition [${a.show}] at position $position")
```
Comment (dilyand) [inline loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/AtomicDef.scala:236]: `position $position` is good, but in all the other cases it's `$position position` which I think is less clear.

File: loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/package.scala [hunk 4]
```diff
@@ -52,10 +52,12 @@ package object ast {
 
   implicit object CastedColumnShow extends Show[Select.CastedColumn] {
```
Comment (dilyand) [inline loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/package.scala:53]: Hate to be a grammar nazi but this should be a `CastColumn` (cast is the same in all three forms).

**Review by dilyand — APPROVED** [review]
Great stuff @chuwy !👍🏻

File: core/src/main/scala/com.snowplowanalytics.snowflake.core/RunManifests.scala (hunk 2 above)
Comment (chuwy) [reply core/src/main/scala/com.snowplowanalytics.snowflake.core/RunManifests.scala:1]: That's easy! 

Scala 2.12 required to Scala Analytics SDK to 2.0.0 because it's not published for 2.11. Scala Analytics SDK 2.0.0 removed `manifest` module, but part of it is still used in Snowflake Loader.

File: loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/AtomicDef.scala (hunk 3 above)
Comment (chuwy) [reply loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/AtomicDef.scala:236]: Agreed.

File: loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/package.scala (hunk 4 above)
Comment (chuwy) [reply loader/src/main/scala/com.snowplowanalytics.snowflake.loader/ast/package.scala:53]: Will address this in the next release https://github.com/snowplow-incubator/snowplow-snowflake-loader/issues/127
